util/reggen/gen_fpv.py

```diff
@@ -0,0 +1,51 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+# # Lint as: python3
+#
+"""Generate FPV CSR read and write assertions from validated register JSON tree
+"""
+"""TODO(chencindy): DO NOT SUBMIT without one-line documentation for gen_fpv.
+
+TODO(chencindy): DO NOT SUBMIT without a detailed description of gen_fpv.
```

Comment:
do you plan to resolve these TODO's in the current PR?

Reply:
Yes I will add it once everyone is okay with this flow.

Reply:
Remove name here. just `TODO: XXX`

Reply:
thanks Eunchan, I think this is automatically generated when I created the file. I will remove the name.